dcfs/dcfs-share/dcfs-run/demo.py

```diff
@@ -398,45 +399,66 @@ def generate_phoenix_autotimestamp():
         wf.write(task_info['hds']['sql'])
 
 df_joined.to_csv(tmp_csv_path, index=False, header=False)
-''' ========== Phoenix ========== '''
+
+''' ========= Store data into HDS without phoenix =========='''
+import requests
+import random
 import subprocess
-phoenix_home = "/opt/phoenix-hbase-2.3-5.1.2-bin"
-if 'ip' in task_info['hds']:
-    hds_ip = task_info['hds']['ip']
-else:
-    hds_ip = 'zoo1'
-try:
-    logging.info('Start importing table into HDS')
-    send_task_status(task_id, TASKSTATUS_PROCESSING, "Start importing table into HDS")
-    cmd = phoenix_home+"/bin/psql.py %s -t \"%s\" %s %s" % (hds_ip, table_name.upper(), tmp_sql_path, tmp_csv_path)
-    process = subprocess.Popen(cmd, shell=True, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
-    stdout, stderr = process.communicate()
-    exit_code = process.wait()
-    stdout = stdout.decode('utf-8')
-    stderr = stderr.decode('utf-8')
-    logging.info('Finished importing table into HDS')
-    send_task_status(task_id, TASKSTATUS_PROCESSING, "Finished importing table into HDS")
-except Exception as e:
-    logging.error("Failed when importing table into HDS\n" + str(e))
-    send_task_status(task_id, TASKSTATUS_FAILED, "Failed when importing table into HDS\n" + str(e))
-    exit(1)
 
-logging.debug("Phoenix stdout\n" + stdout)
-logging.debug("Phoenix stderr\n" + stderr)
-logging.debug(f"Phoenix exit code: {exit_code}")
-if exit_code != 0:
-    logging.error("Failed to import table into HDS\n" + stderr)
-    send_task_status(task_id, TASKSTATUS_FAILED, "Failed to import table into HDS\n" + stderr)
-    exit(1)
-else:
-    logging.error("Successfully importing table into HDS" + stderr)
-    send_task_status(task_id, TASKSTATUS_PROCESSING, "Successfully importing table into HDS")
-''' ========== Phoenix ========== '''
-
-if stderr.find("ERROR") == -1:
-    logging.error("Job finished")
-    send_task_status(task_id, TASKSTATUS_SUCCEEDED, "Job finished")
-else:
-    logging.error("Job finished with error message: \n" + stderr)
-    send_task_status(task_id, TASKSTATUS_SUCCEEDED, "Job finished with error message: \n" + stderr)
+if task_dict['phoenix']=='false':
+    try:
+        send_task_status(task_id, TASKSTATUS_PROCESSING, "Start importing csv file into HDS", '')
+        logging.info("Start importing csv file into HDS")
+        url = "http://hbase-regionserver1:8000/dataservice/v1/access"
+        params = {
+                'from': 'local:///',
+                'to':   'hds:///csv/join/'+'joined_' + task_id + '_' + ts + '.csv'
+        }
+        r = requests.post(url, params=params, data=open(tmp_csv_path, 'rb') , timeout=10)
+        send_task_status(task_id, TASKSTATUS_SUCCEEDED, "Finished importing csv file into HDS", '/dataservice/v1/access?from=hds:///csv/join/joined_' + task_id+'_'+ts +'.csv&to=local:///result.csv')
```

Comment:
Suggest to add `redirectfrom=NULL` to prevent redirecting to the hostname that our clients don't support.

```suggestion
        send_task_status(task_id, TASKSTATUS_SUCCEEDED, "Finished importing csv file into HDS", '/dataservice/v1/access?from=hds:///csv/join/joined_' + task_id+'_'+ts +'.csv&to=local:///result.csv&redirectfrom=NULL')
```